src/net-questdb-client/Senders/ISender.cs

```diff
@@ -32,7 +33,14 @@ namespace QuestDB.Senders;
 /// <summary>
 ///     Interface representing <see cref="Sender" /> implementations.
 /// </summary>
-public interface ISender : IDisposable
+public interface ISender : ISenderV2
```

Comment:
Why do we need ISenderV2? What is the purpose of the split in V1 and V2, I cannot see anything that supports V1 only.